src/top-bar/context-select/context-select.js

```diff
@@ -0,0 +1,80 @@
+import {
+    IconChevronDown24,
+    IconChevronUp24,
+    colors,
+    Popover,
+    Tooltip,
+} from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React, { useRef } from 'react'
+import classes from './context-select.module.css'
+
+const ContextSelect = ({
+    children,
+    label,
+    value,
+    onClose,
+    onOpen,
+    disabled,
+    open,
+    tooltipContent,
+}) => {
+    const buttonRef = useRef()
+    const Icon = open ? IconChevronUp24 : IconChevronDown24
+    const button = (
+        <button
+            ref={buttonRef}
+            className={classes.button}
+            onClick={onOpen}
+            disabled={disabled}
+        >
+            <span className={classes.prefix}>{label}</span>
```

Comment:
Minor: I think the class name and the content variable's name should be in sync (is this a prefix or a label?)